src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC, useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[] | []>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isNoPeople, setIsNoPeople] = useState(false);
```

Comment:
redundant state - you always can check `peopleFromServer.length`
general rule no need to create a state for some value if it might be calculated from another state value/values

Reply:
https://github.com/mate-academy/react_people-table-basics/pull/860/files#r1269812293
not fixed this comment.